### ISSUE TITLE
[BottomSheet] NavigationView in BottomSheetDialog is thick color

### ISSUE THREAD

**Issue by manabu-nakamura** [material-components/material-components-android#3064]
**Description:** The NavigationView in the BottomSheetDialog is thick color because android:windowElevation is 16dp in its theme (ThemeOverlay.Material3.BottomSheetDialog).
![Screenshot_20221102_123629](https://user-images.githubusercontent.com/71050561/200094501-9c9dc9a1-98bd-4f34-a0f5-6d18ed555318.png)

**Expected behavior:** Please put &lt;item name="android:windowElevation"&gt;0dp&lt;/item&gt; in ThemeOverlay.Material3.BottomSheetDialog or its parent.
![Screenshot_20221105_094853](https://user-images.githubusercontent.com/71050561/200094521-2916f552-5784-4766-8b8e-29390ff6eae8.png)

**Source code:**
```
<LinearLayout
    xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    xmlns:tools="http://schemas.android.com/tools"
    android:layout_width="match_parent"
    android:layout_height="wrap_content"
    android:orientation="vertical"
    tools:viewBindingIgnore="true">
    <com.google.android.material.bottomsheet.BottomSheetDragHandleView
        android:layout_width="match_parent"
        android:layout_height="wrap_content" />
    <com.google.android.material.navigation.NavigationView
        android:layout_width="match_parent"
        android:layout_height="wrap_content"
        app:menu="@menu/sheet" />
<!--        android:background="@android:color/transparent"-->
</LinearLayout>
```
```
BottomSheetDialog bottomSheetDialog = new BottomSheetDialog(requireContext());
bottomSheetDialog.setContentView(R.layout.sheet);
bottomSheetDialog.setDismissWithAnimation(true);
//bottomSheetDialog.getWindow().setElevation(0);
EdgeToEdgeUtils.applyEdgeToEdge(bottomSheetDialog.getWindow(), true);
ViewCompat.setOnApplyWindowInsetsListener(bottomSheetDialog.getWindow().getDecorView(), (v, insets) -> insets);
```
https://github.com/material-components/material-components-android/blob/master/lib/java/com/google/android/material/bottomsheet/BottomSheetDialog.java#L388
https://github.com/material-components/material-components-android/blob/master/lib/java/com/google/android/material/theme/res/values/themes_base.xml#L81
https://github.com/material-components/material-components-android/blob/master/lib/java/com/google/android/material/bottomsheet/res/values/themes.xml#L20

**Minimal sample app repro:** [Editor7.zip](https://github.com/material-components/material-components-android/files/9942661/Editor7.zip)

**Android API version:** 32

**Material Library version:** 1.8.0-alpha02

**Device:** Emulator 31.3.13

**Comment by manabu-nakamura** [comment]
The NavigationView in the SideSheetDialog is thick color, too.

**Comment by drchen** [comment]
I feel this is working as intended - the elevation-based surface colors are supposed to be based on the total elevation. 

This behavior may change, though. As I said we are figuring out a new surface color mechanism irrelevant to elevation. So let me close this issue as WAI.

**Comment by manabu-nakamura** [comment]
I think this is a bug. I hope it gets fixed eventually.

**Comment by drchen** [comment]
The elevation of components will be added up.

From the design concept it makes sense to use different surface colors because they indeed have different elevations - imagine the case that one container is used inside another container - this guarantee that they will have different colors.

But as I said - we are planning to discard this elevation-based surface color system. So this issue will be gone when we migrate to that. And in the current system - I think it's WAI.

**Comment by manabu-nakamura** [comment]
Sorry. Could you tell me what WAI stands for?

**Comment by imhappi** [comment]
It means Working As Intended

**Comment by manabu-nakamura** [comment]
@imhappi, thank you.

**Comment by manabu-nakamura** [comment]
This seems to have been fixed in 1.11.0.
